secrets.json

```diff
@@ -1,10 +1,10 @@
 {
   "EXPRESS_SESSIONS_SECRET": "sample-session-secret",
   "ADMIN_SECRET":"CodingBlocks",
-  "DATABASE" : "hb",
-  "DB_USER" : "cbuser",
-  "DB_PASSWORD" : "cbpass",
-  "DB_HOST" : "localhost",
+  "DATABASE" : "daemffcvaov93s",
```

Comment:
don't do it this way. 
take the Heroku DATABASE_URL in sequelize if it's available, or take it from this file.